src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -1167,6 +1269,54 @@ def _copy_scm_credentials(self):
             "Control eda-credentials"
         )
 
+    def _update_postgres_credentials(self):
+        cred_type = models.CredentialType.objects.filter(
+            name=enums.DefaultCredentialType.POSTGRES
+        ).first()
+        inputs = {
+            "postgres_db_host": settings.ACTIVATION_DB_HOST,
+            "postgres_db_port": settings.DATABASES["default"]["PORT"],
+            "postgres_db_name": settings.DATABASES["default"]["NAME"],
+            "postgres_db_user": settings.DATABASES["default"]["USER"],
+            "postgres_db_password": settings.DATABASES["default"]["PASSWORD"],
+            "postgres_sslmode": settings.PGSSLMODE,
```

Comment:
This should be taken from DATABASES: https://github.com/ansible/eda-server/blob/d7fd30ca98545520faff1df2f95161508c310610/src/aap_eda/settings/default.py#L260

Otherwise it could lead to confusion or not honor what it is provided in the settings.yaml (potentially breaking the RPM installer)
_DEFAULT_PG_NOTIFY_DSN_SERVER has the same issue. 

We should change at least this and later we can treat separately the other things that were discussing.